### DISCUSSION TITLE
Missing Code in SQL Databases Tutorial FastAPI Documentation

### DISCUSSION THREAD

**Discussion by starryknight64** [fastapi/fastapi#12393] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Open a browser and goto:
# https://fastapi.tiangolo.com/tutorial/sql-databases/?h=sessionlocal#create-initial-pydantic-models-schemas
# Notice that there are no code samples in the tabbed sections below it
# More info in description...
```

### Description

1. Open the Browser and goto https://fastapi.tiangolo.com/tutorial/sql-databases/?h=sessionlocal#create-initial-pydantic-models-schemas
2. Notice that there are no code samples in the tabbed sections below it
3. Notice that the Tip's refer to code that isn't there. 
4. Perhaps this was mistakenly removed at some point?

### Operating System

Windows

### FastAPI Version

0.115.0

### Pydantic Version

2.9.2

### Python Version

3.12.7

**Comment by JavierSanchezCastro** [comment, marked as answer]
Hello!

Opened PR https://github.com/fastapi/fastapi/pull/12395

**Comment by SteamedTIGER** [comment]
hello
